### DISCUSSION TITLE
Welcome to fantomCX Discussions!

### DISCUSSION THREAD

**Discussion by MikeHart66** [MikeHart66/fantomCX#1] [General]
## 👋 Welcome!
 I use Discussions as a place to connect with other members of my community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hardcoal22** [comment]
Hi

**Reply by MikeHart66** [reply]
Hello again!

**Comment by hardcoal22** [comment]
You put so much effort on your framework.. 
how come you dont have a website ..dedicated?
You can find free websites this days.. 

like mine.. http://www.hcgameeditor.dx.am/

Anyway.. Im gonna see where im going with this.. 
and if ill succeed.  Ill make some games and post them online.

working on physics now..

**Reply by MikeHart66** [reply]
Back in the day when Monkey was actually a thing, I had my own website under fantomgl.com. But this is past time. Now it isn't needed anymore. Webspace and domains are certainly not an issue.
